### PR TITLE
Added drone.yml

### PR DESCRIPTION
**Pull request by gal**

### CODE REVIEW

**Review by ericm — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+volumes:
+- name: cache
+  temp: {}
+
+steps:
+- name: build
+  image: plugins/docker
+  settings:
+    tag: latest
+    repo: docker.netsoc.co/public/netsoc.co_new
```
Comment (Strum355) [inline .drone.yml]: do we wanna just overwrite the whole website? :man_shrugging:

Comment (gal) [reply .drone.yml]: Eventually, yes I would imagine, but for the time being, keep them separate until full release of new website, and maybe archive the future "old" website?

Comment (ericm) [reply .drone.yml]: I mean it's an easy change when the websites working

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      - latest
+    repo: docker.netsoc.co/public/netsoc.co
+    registry: docker.netsoc.co
+    username:
+      from_secret: docker_dev_username
+    password:
+      from_secret: docker_dev_auth
+  volumes:
+  - name: cache
+    path: /var/lib/docker
```
Comment (Strum355) [inline .drone.yml]: This and the volumes part on L4 can be deleted, serve  no purpose here

**Review by Strum355 — CHANGES_REQUESTED** [review]

File: .drone.yml [hunk 3]
```diff
@@ -1,70 +1,63 @@
...
+    settings:
+      username: Netsoc CI
+      avatar_url: https://raw.githubusercontent.com/drone/brand/3051b0d85318a2a20b62927ba19fc07e24c0d751/logos/png/white/drone-logo-png-white-256.png
+      color: "#e04414"
+      message: netsoc.co failed for {{build.author}}. Build num {{build.number}}. {{build.link}}
+    when:
+      event:
+        - push
+      branch:
+        - master
```
Comment (Strum355) [inline .drone.yml]: should this be new_website?

File: .drone.yml [hunk 4]
```diff
@@ -1,70 +1,63 @@
...
-    event:
-      - push
-    branch:
-      - new_website
+  - name: build
+    image: plugins/docker
+    settings:
+      tags:
+        - "$(DRONE_COMMIT_SHA:0:7)"
+        - latest
```
Comment (Strum355) [inline .drone.yml]: if we're leaving things on new_website branch for now, will we use `new` tag so it doesnt overwrite current website latest tag? We'll still have a commit-hash tagged image for the old website, but atm things point to latest

File: .drone.yml (hunk 3 above)
Comment (ericm) [reply .drone.yml]: Yeah it should

File: .drone.yml (hunk 4 above)
Comment (ericm) [reply .drone.yml]: `new` should be grand

File: .drone.yml [hunk 5]
```diff
@@ -0,0 +1,88 @@
+kind: pipeline
+name: default
+
+steps:
+  - name: build
+    image: plugins/docker
+    settings:
+      tags:
+        - '$(DRONE_COMMIT_SHA:0:7)'
```
Comment (Strum355) [inline .drone.yml]: curly braces : )

**Review by Strum355 — CHANGES_REQUESTED** [review]
build step references docker_dev_username and docker_dev_auth, but theyre defined as registry_pub_token and registry_pub_user :)

**Review by Strum355 — APPROVED** [review]

**Pull request merged**